atlp-rwanda/atlp-devpulse-fn
# #147 Implementing google authentication

### PR Description
This PR Allows users to Login/Signup using Google authentication

### Description of tasks that were expected to be completed
- Fix the google login button
- Create an additional page google users go through to add more of their data

### How has this been tested?
- Clone repo
- Checkout branch `fix-google-auth`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully

### Credentials
CLIENT_ID: (Ask devops)

### Track PR (issue number & link)
[#147 ](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/147)

### Screenshots

1.Login page button working click and choose an account
![Screenshot 2024-10-11 140045](https://github.com/user-attachments/assets/d8d1d7f7-5816-4caf-87de-08c837389774)
![Screenshot 2024-10-11 114751](https://github.com/user-attachments/assets/677b4110-f019-4dda-bb37-4eabe138deb9)



2. If it's a user's first time using google authentication
![Screenshot 2024-10-11 114814](https://github.com/user-attachments/assets/7b33d254-df3e-4132-baa0-f218a3a5d212)



3.After filling in the data the user is created successfully you can now access the rest of the application
![Screenshot 2024-10-11 114840](https://github.com/user-attachments/assets/de1d918a-b795-4d35-a8a0-69cd35dc55f9)
